assets/sass/components/global/_googlesitekit-gathering-data-notice.scss

```diff
@@ -70,3 +70,20 @@
 		}
 	}
 }
+
+#wpadminbar {
+
+	.googlesitekit-plugin {
+
+		.googlesitekit-gathering-data-notice {
+
+			p {
+
+				color: $c-jumbo;
+				font-size: 11px;
+				line-height: 13px;
+				text-transform: lowercase;
```

Comment:
I'm not sure why these are lowercased in the design, because generally using all lowercase or all uppercase is problematic for accessibility (especially users with dyslexia).

Probably not a huge deal here and something we should tackle in all of our components, but just calling it out for reference here.

Reply:
It's a fair comment. For reference, the `p` is already lowercased elsewhere - it just proved impractical to reuse these rules, hence the `#wpadminbar`-scoped addition.

https://github.com/google/site-kit-wp/blob/13f84ebb3824455d2f7ffad38ef301c6c67df523/assets/sass/components/global/_googlesitekit-gathering-data-notice.scss#L25-L31